rclcpp/include/rclcpp/client.hpp

```diff
@@ -676,7 +711,7 @@ class Client : public ClientBase
     PromiseWithRequest promise;
     auto shared_future = promise.get_future().share();
     auto req_id = async_send_request_impl(
-      *request,
+      request,
```

Comment:
why not `std::move` as in the other calls above?

Reply:
Not moving here because needs to be valid for later use in line 718